### DISCUSSION TITLE
Looking for contributors

### DISCUSSION THREAD

**Discussion by maifeeulasad** [maifeeulasad/to-spreadsheet#3] [Announcements]
Simply looking for contributors

**Comment by maifeeulasad** [comment]
@Qaplagzy, @shuanhukexuejia, @kilik52 are you available? It would be really helpful and speed up the development process.

**Reply by Qaplagzy** [reply]
yeah, sure, how's it going?

**Reply by shuanhukexuejia** [reply]
sure, need some help?

**Reply by maifeeulasad** [reply]
Okay, here is the thing. On the latest release, I'm able to generate an excel sheet. And the worksheet generation works fine. Now there are lots of parameters to take care of, and I'm feeling a bit exhausted. So it would be helpful if we could simply split some work and do it.

Sorry for the late reply, got stuck in another project for a while. Now I'm free. Thanks.

**Reply by Qaplagzy** [reply]
Nice work. We will spend about half a day reviewing your code. Can you give us a to-do list?

**Reply by maifeeulasad** [reply]
@Qaplagzy, @shuanhukexuejia can you guys start reviewing it?

I still don't have a todo list. So feel free to review and and give us some insight maybe. 

Latest release (https://github.com/maifeeulasad/to-spreadsheet/tree/v1.0.0) can generate XLSX properly, although lots of features are to be done.

I will start creating a roadmap and small issues for them.
